Source/Core/DiscIO/CISOBlob.cpp

```diff
@@ -44,7 +44,7 @@ CISOFileReader* CISOFileReader::Create(const std::string& filename)
 
 u64 CISOFileReader::GetDataSize() const
 {
-	return GetRawSize();
+	return CISO_MAP_SIZE * m_block_size;
```

Comment:
But what if m_block_size does have a different value in some file? I don't see what the advantage of a hardcoded magic number would be.